src/lib_json/json_reader.cpp

```diff
@@ -54,7 +54,7 @@ static size_t const stackLimit_g = JSONCPP_DEPRECATED_STACK_LIMIT; // see readVa
 
 namespace Json {
 
-#if __cplusplus >= 201103L || (defined(_CPPLIB_VER) && _CPPLIB_VER >= 520)
+#if __cplusplus >= 201103L || (defined(_CPPLIB_VER) && _CPPLIB_VER >= 520) && (_GLIBCXX_USE_CXX11_ABI > 0)
```

Comment:
WTF? Now I have `defined(_CPPLIB_VER) && _CPPLIB_VER >= 520 && _GLIBCXX_USE_CXX11_ABI > 0` what is never the case. I don't have time for this.